packages/shared-ux/page/solution_nav/src/with_solution_nav.scss

```diff
@@ -1,17 +1,8 @@
+// TODO: Can now be converted to Emotion
```

Comment:
We should update https://github.com/elastic/kibana/issues/122594 and reference it here.